src/test/java/com/github/tomakehurst/wiremock/BindAddressTest.java

```diff
@@ -78,23 +79,16 @@ public void stop() {
   @Test
   public void shouldRespondInTheBindAddressOnlyOnHttp() throws Exception {
     executeGetIn(localhost);
-    try {
-      executeGetIn(nonBindAddress);
-      fail("Should not accept HTTP connection to [" + nonBindAddress + "]");
-    } catch (Exception ex) {
-    }
+
+    assertThrows(RuntimeException.class, () -> executeGetIn(nonBindAddress));
```

Comment:
We've changed the intent of this test, right? Should be:

```suggestion
    assertThrows(Exception.class, () -> executeGetIn(nonBindAddress));
```

To match the previous intent?